libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -441,7 +713,7 @@ bool ASTJsonConverter::visit(Literal const& _node)
 	if (!dev::validateUTF8(_node.value()))
 		value = Json::nullValue;
 	Token::Value subdenomination = Token::Value(_node.subDenomination());
-	addJsonNode(_node, "Literal", {
+	setJsonNode(_node, "Literal", {
 		make_pair("token", tokenString ? tokenString : Json::Value()),
 		make_pair("value", value),
 		make_pair("hexvalue", toHex(_node.value())),
```

Comment:
I think it is more important to keep backwards compatibility here. Note that this is an extremely deep interface and everything has to be translated into another internal representation anyway - it is not an API you use multiple times.